sgv2-docsapi/README.md

```diff
@@ -0,0 +1,236 @@
+| `stargate.token-resolver.fixed.token`        | `String` | unset               | Tenant identifier value if the `fixed` type is used.                                                                       |
+
+## Development guide
+
+This project uses Quarkus, the Supersonic Subatomic Java Framework.
+If you want to learn more about Quarkus, please visit its [website](https://quarkus.io/).
+
+It's recommended that you install Quarkus CLI in order to have a better development experience.
+See [CLI Tooling](https://quarkus.io/guides/cli-tooling) for more information.
+
+Note that this project uses Java 17, please ensure that you have the target JDK installed on your system.
+
+### Running the application in dev mode
+
+You can run your application in dev mode that enables live coding using:
+```shell script
+./mvnw compile quarkus:dev
+```
+
+> **_NOTE:_**  Quarkus now ships with a Dev UI, which is available in dev mode only at http://localhost:8180/stargate/dev/.
+
+#### Debugging
+
+In development mode, Quarkus starts by default with debug mode enabled, listening to port `5005` without suspending the JVM.
+You can attach the debugger at any point, the simplest option would be from IntelliJ using `Run -> Attach to Process`.
+
+If you wish to debug from the start of the application, start with `-Ddebug=client` and create a debug configuration in a *Listen to remote JVM* mode.
+
+See [Debugging](https://quarkus.io/guides/maven-tooling#debugging) for more information.
+
+### Packaging and running the application
+
+The application can be packaged using:
+```shell script
+./mvnw package
+```
+It produces the `quarkus-run.jar` file in the `target/quarkus-app/` directory.
+Be aware that it’s not an _über-jar_ as the dependencies are copied into the `target/quarkus-app/lib/` directory.
+
+The application is now runnable using `java -jar target/quarkus-app/quarkus-run.jar`.
+
+If you want to build an _über-jar_, execute the following command:
+```shell script
+./mvnw package -Dquarkus.package.type=uber-jar
+```
+
+The application, packaged as an _über-jar_, is now runnable using `java -jar target/*-runner.jar`.
+
+### Creating a native executable
+
+You can create a native executable using: 
+```shell script
+./mvnw package -Pnative
+```
+
+Or, if you don't have GraalVM installed, you can run the native executable build in a container using: 
+```shell script
+./mvnw package -Pnative -Dquarkus.native.container-build=true
+```
+
+You can then execute your native executable with: `./target/sgv2-docsapi-${project.version}-runner`
+
+If you want to learn more about building native executables, please consult [Maven tooling](https://quarkus.io/guides/maven-tooling).
+
+### Creating a docker image
+
+You can create a docker image named `io.stargate/docsapi` using:
```

Comment:
@jeffreyscarpenter What would our SGv2 naming convention be, wrt existing REST API docker image?

Reply:
I checked, you have `io.stargate/restapi`, so I followed the same :)

Reply:
Good, good, just wanted to mention!